windows/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+target_sources(ninja PRIVATE 
+    ninja.manifest
+    ninja.rc.in
+)
+
+configure_file(ninja.rc.in ninja.rc @ONLY)
```

Comment:
The @ONLY will replace variables with the @NINJA_VERSION_MAJOR@ with the equivalent CMake variable